app/models/suspended_email.rb

```diff
@@ -16,11 +16,11 @@ def create_from_email_address!(email_address)
     end
 
     def find_with_email(email)
-      find_by(digested_base_email: generate_email_digest(email))&.email_address
+      find_by(digested_base_email: generate_email_digest(email))
     end
 
     def find_with_email_digest(digest)
```

Comment:
we've changed the return type of this method (and the one above), I'd consider adding a comment to help make it clear

```suggestion
    # @return [SuspendedEmail,nil]
    def find_with_email_digest(digest)
```